tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -467,21 +467,35 @@ def test_show_queue_counters(self, setup, setup_config_mode, duthosts, enum_rand
 
         for key in buffer_queue_keys:
             try:
-                interfaces.add(key.split('|')[1])
+                # The interface name is always the last but one field in the BUFFER_QUEUE entry key
+                interfaces.add(key.split('|')[-2])
             except IndexError:
                 pass
 
+        # For the test to be valid, we should have at least one interface selected
+        assert (len(interfaces) > 0)
+
+        intfsChecked = 0
         if mode == 'alias':
             for intf in interfaces:
-                alias = setup['port_name_map'][intf]
+                alias = setup['port_name_map'].get(intf)
+                if not alias:
+                    continue
```

Comment:
Please make this specific to voq chassis. on non-voq systems, every interface is expected to have alias returned

Reply:
ack. I was not sure of what assumptions to make here. I can add a "voq chassis" check here (and in the other location mentioned below).

Reply:
fixed in latest.